### PR TITLE
Fix BlurredLoader not working properly.

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
As the title says.

#### What problem is this solving?
BlurredLoad start after the image has loaded. You can check this behavior [here](https://storecomponents.myvtex.com/)

#### How should this be manually tested?
[Access this workspace](https://blurredimage--storecomponents.myvtex.com/)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductImages/styles.css [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
+}
 
-.image { }
+.image {
+}
 
-.imageBlur30 { 
-  filter: blur(30px);
+.imageBlur30 {
+  content: ‘’;
```
Comment (GustavoMA) [inline react/components/ProductImages/styles.css]:
```suggestion
  content: "";
```
Or
```suggestion
  content: '';
```

File: react/package.json [hunk 2]
```diff
@@ -26,18 +26,19 @@
...
     "recompose": "^0.30.0",
     "render": "^0.1.4",
     "slick-carousel": "^1.8.1",
     "underscore": "^1.9.1",
-    "vtex-tachyons": "^2.3.0",
-    "react-intl": "^2.8.0"
+    "vtex-tachyons": "^2.3.0"
   },
   "devDependencies": {
     "@vtex/test-tools": "^0.1.2",
```
Comment (GustavoMA) [inline react/package.json]: Could you do a favor and remove the 
```
  "vtexTestTools": {
    "defaultLocale": "en"
  }
```
It's not necessary anymore :)

**Comment by gabrielgalc** [comment]
We're getting close, but the image is still blinking twice (sometimes it crops in half, sometimes it displays the gray background again). It's an inconsistent error. It should be a smooth experience like this. Sorry I'm being so demanding on this, but these kind of details is what makes our components great and irreplaceable

File: react/components/ProductImages/components/BlurredLoader/Loader.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          </div>
+        </div>
+      </div>
+    </animated.div>
+  )
+}
+
+Loader.propTypes = {
+  /** Loader Type(Spinner or Linear Progress) */
+  loaderType: PropTypes.string,
```
Comment (lbebber) [inline react/components/ProductImages/components/BlurredLoader/Loader.js]: I think an enum is in order here (`PropTypes.oneOf`)

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 4]
```diff
@@ -23,136 +18,89 @@ export const LOADER_TYPES = {
...
-            />
-          </div>
-        )}
       </div>
-    )
-  }
+      <div className="h-100 w-100">
+        <Loader loaded={loaded} loaderType={loaderType}>
+          <ImageResizer
+            alt=""
```
Comment (lbebber) [inline react/components/ProductImages/components/BlurredLoader/index.js]: why this empty alt?

File: react/components/ProductImages/components/Carousel/index.js [hunk 5]
```diff
@@ -133,6 +136,7 @@ class Carousel extends Component {
 
     if (!equals(prevProps.slides, this.props.slides)) {
       this.setInitialVariablesState()
+
```
Comment (lbebber) [inline react/components/ProductImages/components/Carousel/index.js]: 🤷‍♂️

File: react/components/ProductImages/styles.css [hunk 6]
```diff
@@ -24,18 +36,58 @@
...
+  height: 5px;
+  overflow-x: hidden;
+}
+
+.subline {
+  position: absolute;
+  height: 5px;
+}
+.inc {
+  animation: increase 2s infinite;
```
Comment (lbebber) [inline react/components/ProductImages/styles.css]: what is the purpose of these animations? animating `left` or `width` is generally not good for performance, it'd be better to stick with `transform` as much as possible

File: react/components/ProductImages/components/BlurredLoader/index.js (hunk 4 above)
Comment (JuanAlmeida12) [reply react/components/ProductImages/components/BlurredLoader/index.js]: Ops, I don't see this. Sorry.

File: react/components/ProductImages/styles.css (hunk 6 above)
Comment (JuanAlmeida12) [reply react/components/ProductImages/styles.css]: I use this to animate the linear loader.

**Pull request merged**